Modules/Scripted/DICOMPlugins/DICOMScalarVolumePlugin.py

```diff
@@ -224,10 +230,40 @@ def examineFiles(self, files):
             for tag in subseriesTags:
                 value = slicer.dicomDatabase.fileValue(file, self.tags[tag])
                 value = value.replace(",", "_")  # remove commas so it can be used as an index
+
                 if tag not in subseriesValues:
                     subseriesValues[tag] = []
-                if not subseriesValues[tag].__contains__(value):
+
+                if tag == "imageOrientationPatient" or tag == "diffusionGradientOrientation":
+                    if value != "":
```

Comment:
A comment here to define why these tags are unique would help.  Also defining a list or set with the name like `vectorTags` then checking `if tag in vectorTags:` would make the code more readable and easier to cleanly extend to more vector tags in the future.

Reply:
I agree